### ISSUE TITLE
Minor bugs

### ISSUE THREAD

**Issue by davepascual** [crocodic-studio/crudbooster#1065]
- Operating system (name, version and architecture)
Windows 7 Ultimate x64 
- Web server (name, version and architecture)
Apache 2.4.27
- Browser (name, version and architecture)
Google Chrome Version 64.0.3282.186 (Official Build) (32-bit)
- CRUDBooster version
v5.4.0.15
- Laravel version
5.6.8
- What is the problem?
Not a major bug.
I use Super Admin account when I encountered these.
When I update my name in Users Management it does not reflect to my current information listed on the dashboard, but in the user details it does. Then I logged out and logged in again and it updates my information in dashboard. 
When changing other user's color theme it also changes my color theme too even I don't change mine.
- How should it happen? What was the expected behavior?
- Tell how to reproduce the problem step by step
- If possible, take screenshots or send snippets of code

**Comment by rallisf1** [comment]
Yes the problem is because of the session. Add a hook_after_edit & hook_after_delete to your AdminCmsUsersController and delete the session for the edited/deleted user. This will force logout the user, but that is the right thing to do anyway.
